docs/guides/pagination/overview.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Pagination - Overview
+---
+
+# Pagination
+
+Whenever we deal with lists in GraphQL, we usually need to limit the number of items returned. Surely, we don't want to send massive lists of
+items that take a considerable toll on the server! The goal of this guide is to help you get going fast with pagination!
+
+## Pagination at a glance
+
+We have always dealt with pagination in different situations. Let us take a look at some of the common ways pagination
+can be implemented today!
+
+### Offset based pagination
+
+This pagination style is similar to the syntax we use when looking up database records. Here, the client specifies the number of result to be
+obtained at a time, along with an offset- which usually denotes the number of results to be skipped from the beginning. This type of pagination
+is widely used. Implementing offset-based pagination with an SQL database is straight-forward:
```

Comment:
```suggestion
is widely used. Implementing offset-based pagination with a SQL database is straight-forward:
```
This might start some holy wars :grin: